qutebrowser/config/configtypes.py

```diff
@@ -1375,14 +1399,6 @@ def validate(self, value):
         elif 'always' in values and len(values) > 1:
             raise configexc.ValidationError(
                 value, "List cannot contain always!")
-        # Values have to be valid
-        elif not set(values).issubset(set(self.valid_values.values)):
-            raise configexc.ValidationError(
-                value, "List contains invalid values!")
-        # List can't have duplicates
-        elif len(set(values)) != len(values):
-            raise configexc.ValidationError(
-                value, "List contains duplicate values!")
 
     def complete(self):
```

Comment:
Hm sure, I wasn't sure if this would get too confusing with too many combinations, but you can always restrict `combinable_values`.